EnglishFilter/sections/specific.txt

```diff
@@ -259,6 +259,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+factinate.com#$#.advertisment_notice { position: absolute !important; left: -3000px !important; }
+factinate.com#$#.home-page-ad { position: absolute !important; left: -3000px !important; }
```

Comment:
Please move these rules to `general_extensions.txt`, to this section:
https://github.com/AdguardTeam/AdguardFilters/blob/b9e391850a6559a6d44540fad707d63182736465/EnglishFilter/sections/general_extensions.txt#L3194-L3199
With comment like in other rules, I mean:
```
! https://github.com/AdguardTeam/AdguardFilters/issues/77107
factinate.com#$#.advertisment_notice { position: absolute !important; left: -3000px !important; }
factinate.com#$#.home-page-ad { position: absolute !important; left: -3000px !important; }
```